controller.js

```diff
@@ -17,13 +17,18 @@ returnTopics()
 
 function getArticle(req, res){
     returnArticle()
-
+}
 
 function getApi(req, res){
     res.status(200).send(data)
 }
 
+function getArticles(req,res){
+    returnArticles()
```

Comment:
I'd be careful about the naming of your model as it is almost identical to the single article model which might lead to bugs in the future if someone was skim reading your code. Same goes for the controller name